src/com/mambu/apisdk/model/ApiGLJournalEntry.java

```diff
@@ -0,0 +1,46 @@
+package com.mambu.apisdk.model;
+
+import com.mambu.accounting.shared.model.EntryType;
+
+/**
+ * A wrapper class to support Posting GLJournalEntries API. This API requires specifying an entry with the GL code and
+ * transaction amount for each debit and credit entry. See MBU-1737.
+ * 
+ * @author mdanilkis
+ * 
+ */
+public class ApiGLJournalEntry {
+	private String glCode; // account's GL code
+	private EntryType entryType; // transaction entry type
+	private String amount; // transaction amount
+
+	public ApiGLJournalEntry(String glCode, EntryType entryType, String amount) {
+		this.glCode = glCode;
+		this.entryType = entryType;
+		this.amount = amount;
+	}
+
+	public void setGlCode(String glCode) {
+		this.glCode = glCode;
+	}
+
+	public void setEntryType(EntryType entryType) {
+		this.entryType = entryType;
+	}
+
+	public void setAmount(String amount) {
```

Comment:
Yes, it does make good sense. I will change the amount to be a BigDecimal instead of a String